migrations/scripts/3_deploy_libs.js

```diff
@@ -0,0 +1,39 @@
+const realm = process.env.WITNET_EVM_REALM || "default"
+const addresses = require("../addresses")[realm]
+const settings = require("../settings")
+
+let Witnet, CBOR
+
+module.exports = function (deployer, network) {
+  try {
+    Witnet = artifacts.require(settings.artifacts[realm].Witnet || settings.artifacts.default.Witnet || "Witnet")
+    CBOR = artifacts.require(settings.artifacts[realm].CBOR || settings.artifacts.default.CBOR || "CBOR")
+  } catch {
+    console.log("Skipped: libs artifacts not found.")
+    return
```

Comment:
I don't know if I understand this logic. If the artifacts are not found, we simply do nothing about it?

Reply:
For instance, it could be the case you were just deploying "npm run migrate-flattened WitnetProxy omgx.rinkeby". As library artifacts are not involved in this process it could happen that "Witnet" and "CBOR" were not previously generated. As we just pretend to migrate the "WitnetProxy", the missing library artifacts could just be ignored and thus, avoid showing a misleading error message to the contract deployer.